src/backend/libpq/rangerrest.c

```diff
@@ -464,6 +466,11 @@ static int call_ranger_rest(CURL_HANDLE curl_handle, const char* request)
 		}
 		else
 		{
+			if (switchToMaster)
+			{
+				/* master's RPS has recovered, switch from standby's RPS to master's RPS */
+				elog(NOTICE, "switch from standby's RPS to master's RPS");
```

Comment:
When this message logged, hawq maybe still talking to standby RPS, right?
example:
```
talk to standby before;
time up, set switchToMaster = true;
talk to master but failed.
retry talk to standby rps successfully
if (switchToMaster) elog(...)
```

may be a bit confusing to user: 
saw log "switch back to master", but still talking to standby now.
looks like elog this is better?
```
"tried to switch back, but %s", talkingWithStandby? "failed":"success";
```

And keep it as it's now is also well.  🙂